api/src/pcapi/core/users/api.py

```diff
@@ -888,6 +889,18 @@ def create_user_access_token(user: models.User) -> str:
     return create_access_token(identity=user.email, additional_claims={"user_claims": {"user_id": user.id}})
 
 
+def create_user_refresh_token(device_info: "account_serialization.TrustedDevice | None", user: models.User) -> str:
```

Comment:
(détail) Puisque le 2ème argument est possiblement vide (on pourrait même dire optionnel ?), j'inverserai l'ordre des arguments. De plus, c'est `user` l'argument le plus important de cette fonction.

```suggestion
def create_user_refresh_token(user: models.User, device_info: "account_serialization.TrustedDevice | None") -> str:
```